docs/metrics/accelerate-changes.md

```diff
@@ -1,5 +1,7 @@
 # Changes and reviews metrics
 
+(WARNING ABOUT BITBUCKET LIMITATIONS?)
```

Comment:
@mrfyda @claudiacodacy Also, I believe we should include a similar business-oriented message about the current Bitbucket integration limitations on the pages describing the metrics that won't be delivered for V1. Do you agree?

Reply:
That makes sense to me! :) we should even consider this inside the platform for v2, perhaps... 🤔

Reply:
@prcr I agree with you, let's keep the limitations centrally on the page dedicated to the Bitbucket integration. People integrating with GitHub, for example, and reading the reference for the other metrics, don't really care about Bitbucket limitations.
I removed the messages from review and changes metrics, and work in progress metrics.